package.json

```diff
@@ -44,7 +45,7 @@
     "lint": "tslint -c tslint.json 'src/**/*.ts' --fix",
     "pretest": "npm run build",
     "postinstall": "npm run build",
-    "test": "npm run lint && jest",
+    "test": "npm run lint && jest --detectOpenHandles",
```

Comment:
Why do we need this flag?

Reply:
Sorry I forgot to remove this. I was having issues with getting jest to connect to database and added it for debugging. I'll remove it